app/content/carto-user-manual/connections/creating-a-connection.md

```diff
@@ -182,3 +182,37 @@ Please visit the documentation of the Databricks [connection parameters](/analyt
 ![Connection setup with Databricks](/img/cloud-native-workspace/connections/the_connections_databricks_parameters.png)
 
 Once you have entered the parameters, you can click the *Connect* button. CARTO will try to connect to your Databricks cluster. If everything is OK, your new connection will be registered.
+
+### IP Whitelisting
+
+CARTO will connect to your databases using a set of static IPs for each region. These static IPs can be whitelisted in your security setups as needed. Here's the list of static IPs, with an endpoint that you can use to validate or retrieve them.
+
+{{% bannerNote title="What is my organization region?" type="note" %}}
+The region (tenant) is specified when creating the organization. Your region is always at the start of the url, for example if you go to your Workspace and the URL is: `https://gcp-us-east1.app.carto.com...`, it means your organization is in `us-east1`.
+{{%/ bannerNote %}}
+
+##### 🇺🇸 us-east1
+You can validate these IP addresses <a href="https://workspace-gcp-us-east1.app.carto.com/meta" target="_blank">here</a>.

```

Comment:
It sounds strange this sentence. 
I would put the list and maybe a link underneath saying something like 

```
Get the list in an <a href="https://workspace-gcp-us-east1.app.carto.com/meta" target="_blank">API Call</a>
```
BTW, why does it need the HTML link and not the markdown one?

Reply:
Because afaik there's no other way to add `target="_blank"` for it to open in a new tab

Reply:
Done!